docs/rationale.rst

```diff
@@ -1,50 +1,111 @@
 Rationale
 #########
 
+*Cliquet* is a toolkit to ease the implementation HTTP `microservices`_.
+Its mainly focused on resource oriented REST APIs (aka :term:`CRUD`).
+
+.. _microservices: http://en.wikipedia.org/wiki/Microservices
+
+
 Philosophy
 ==========
 
-*Cliquet* doesn't try to be a framework: the generated APIs are well defined and
-follow a specific protocol.
+* KISS
+* No magic
```

Comment:
We're sometimes using `;` at the end of the items and sometimes not. We should be consistent.